topostats/filters.py

```diff
@@ -22,43 +21,36 @@ class Filters:
 
     def __init__(
         self,
-        img_path: Union[str, Path],
+        image: np.ndarray,
+        filename: str,
+        pixel_to_nm_scaling: float,
         threshold_method: str = "otsu",
         otsu_threshold_multiplier: float = 1.7,
         threshold_std_dev: float = None,
         threshold_absolute_lower: float = None,
         threshold_absolute_upper: float = None,
-        channel: str = "Height",
         amplify_level: float = None,
         gaussian_size: float = None,
         gaussian_mode: str = "nearest",
-        output_dir: Union[str, Path] = None,
         quiet: bool = False,
     ):
         """Initialise the class.
 
         Parameters
         ----------
-        img_path: Union[str, Path]
-            Path to a valid image to load.
-        channel: str
-            Channel to extract from the image.
+        image: np.ndarray
+            The raw image from the AFM.
+        filename: str
+            The filename (used for logging outputs only).
         amplify_level : float
```

Comment:
I should have added a docstring here for `px_to_nm_scaling` my bad

Reply:
Ah, good spot, added the other missing docstring fields too.